full-node/db/sov-schema-db/src/lib.rs

```diff
@@ -69,12 +69,12 @@ impl DB {
     }
 
     /// Open RocksDB with the provided column family descriptors.
-    pub fn open_cf(
+    fn open_cf(
```

Comment:
I think we should to keep this public. If private, there's no way for a downstream user to open a `DB` with custom `rocksdb::Options` and we're forcing everybody to use the options used by `DB::open`.

Reply:
public `open` also has ` db_opts: &rocksdb::Options,`, this method only provides opportunity to specify column options. 

I understand that it adds another level of configurability, but on the other hand it makes public interface more complicated. I don't have a strong evidence, that somebody might need custom options for column families. But it also does not mean, that this use case does not exist. As they say, absence of evidence is not evidence of absence.

Overall I am fine to keep it public. I will probably open follow up PR, with different approach regarding many public methods

Reply:
Well, apparently we do have a need for custom options (LZ4), so I think it's likely someone else will too, and it'd be nice for them to be able to use the crate. That said, I don't feel too strongly about this!

Reply:
Ok, let's keep it public